src/lib/config.ts

```diff
@@ -55,6 +55,9 @@ export function intVar<R>(varName: string, defaultValue?: R): number | R {
 }
 
 export const API_HOST = requiredVar('API_HOST');
+export const API_HEARTBEAT_STATE_TIMEOUT = intVar(
+	process.env.API_HEARTBEAT_STATE_TIMEOUT || '15',
```

Comment:
It's generally good to convey the expected unit either on the env var (eg. `API_HEARTBEAT_STATE_TIMEOUT_SECONDS`) or at least as a comment next to the number, because things will get messy if it's set to eg. 15000 by someone thinking this is in milliseconds, as is typical with timeouts.

Reply:
Good call.